_config.yml

```
@@ -56,10 +56,18 @@ collections:
     output: true
     permalink: /team/:name
 
+# Configure category pages plugin here
+permalink: /:year/:month/:day/:title:output_ext #gets rid of index.html from category page links (probably)
+category_path: "/blog/category" # category_path: "jekyll/category" # <- default value
+# category_layout: "category_index.html" # <- default value
+paginate: 1 # <- turn on pagination with two posts per page
```

```suggestion
paginate: 2 # <- turn on pagination with two posts per page
```

Right now, the paginator is limiting each page to a single post. When we get more posts, I think it would be better suited to something like 4-5 posts+blurbs per page, just to ensure that the user isn't clicking through too many pages, but things LGTM right now!